src/screens/NationCreateContainer/index.js

```diff
@@ -103,27 +117,52 @@ class NationCreateContainer extends Component {
     });
   };
 
-  _submitForm = () => {
+  submitForm = () => {
     alert('submitForm', [
       {
         name: 'cancel',
         style: 'cancel',
       }, {
         name: 'confirm',
-        onPress: () => this.props.onSubmitNation(this.props.editingNation, () => {
-          if (this.props.latestError) {
-            errorAlert(this.props.latestError);
-            return;
-          }
-
-          this.props.navigator.dismissModal();
-        }),
+        onPress: () => {
+          if (this.props.editingNation === null) return;
+
+          this.props.onSubmitNation(this.props.editingNation, () => {
+            if (this.props.latestError) {
+              errorAlert(this.props.latestError);
+              return;
+            }
+
+            this.props.navigator.dismissModal();
+          });
+        },
       }]);
   };
+
+  render() {
+    return (
+      <NationCreateScreen
+        {...this.props}
+        onCancelNationCreation={this.cancelNationCreation}
+        onResetNationCreation={this.resetNationCreation}
+        onSaveNationDraft={this.saveForm}
+        onDeleteNationDraft={this.deleteForm}
+        onSubmitNation={this.submitForm}
+      />
+    );
+  }
 }
 
-NationCreateContainer.PropTypes = {
-  navigator: PropTypes.object,
+NationCreateContainer.defaultProps = {
+  navigator: null,
```

Comment:
Please remove it, since navigator is required prop.

Reply:
Done